var/spack/repos/builtin/packages/py-pdbfixer/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class PyPdbfixer(PythonPackage):
+    """PDBFixer is an easy to use application for fixing problems in
+    Protein Data Bank files in preparation for simulating them."""
+
+    homepage = "https://github.com/openmm/pdbfixer"
+    url = "https://github.com/openmm/pdbfixer/archive/refs/tags/v1.8.1.tar.gz"
+
+    version('1.8.1', sha256='d50551abfe9dbaefc066f4d9d400cdebe57f1fefd9de9d01e12beb87efd99595')
+    version('1.7', sha256='a0bef3c52a7bbe69a6aea5333f51f3e7d158339be5829aed19b0344bd66d4eea')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('openmm', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('openmm@7.1:', type=('build', 'run'))
```

Reply:
Ah, thanks for reminding me - this is more difficult than that as they moved openmm from simtk.openmm to openmm and things do break if the versions don't line up as expected...